### ISSUE TITLE
data corruption returned by fetchSCAN() queries

### ISSUE THREAD

**Issue by dylanbeaudette** [ncss-tech/soilDB#26]
Every query to the SCAN / SNOTEL web-service returns an "extra" day on September 30th.

![image](https://user-images.githubusercontent.com/624277/29428436-0682c3e6-8342-11e7-96c3-e8dd4714ad7f.png)

**Comment by HONG1992** [comment]
Excuse me？
I'm a novice for R. I would like to use fetchSCAN to query some data but it seems a problem when using this order. I tried to solve this problem by different versions of soilDB packages (2.0-1 and 2.0-3). It didn't work.

> x <- fetchSCAN(site.code=c(356, 2072), year=c(2015,2016))
Error in as.Date.default(x$Date) : 
  do not know how to convert 'x$Date' to class “Date”
> 
> table(x$SMS$depth)
Error in table(x$SMS$depth) : object 'x' not found
> 
> # check the results
> str(x, 1)
Error in str(x, 1) : object 'x' not found

Sorry to disturb you!

**Comment by dylanbeaudette** [comment]
Thanks for reporting this, however, the specific error (a big one) is unrelated to this issue.

Fixed with 7b11d5bb4494695d76a57c9ff1e43b4e5ed83854.

TLDR: SCAN data are now returned from the webservice with an extra line at the top of the file.

You will have to update to the latest version of `soilDB`, then `fetchSCAN()` should work as expected.

I have updated the [tutorial](http://ncss-tech.github.io/AQP/soilDB/fetchSCAN-demo.html).

**Comment by HONG1992** [comment]
Thank you so much.  I appreciate it！
